README.md

```diff
@@ -1,43 +1,73 @@
-cloud-sdk-docker
-================
 
-> :warning: **IMPORTANT NOTE:** The base image for the `cloud-sdk` image will be changed to Alpine Linux on or after 5/15/17.
-> For more information, please see: [#58](https://github.com/GoogleCloudPlatform/cloud-sdk-docker/issues/58)
----
+# Google Cloud SDK Docker
 
-[`google/cloud-sdk`](https://index.docker.io/u/google/cloud-sdk/) is a
-[Docker](https://docker.io) image bundling all the components and dependencies
-of the [Google Cloud SDK](https://cloud.google.com/sdk/) including alpha and
-beta components.
+
+This is Docker image for the [Google Cloud SDK](https://cloud.google.com/sdk/).
+
+Image includes default command line tools:
+*  gcloud:  Default set of gcloud commands
+*  gsutil:  Cloud Storage Command Line Tool
+*  bq: BigQuery Command Line Tool
+
+## Supported tags and respective Dockerfile links
+
+* 155.0.0, latest (Dockerfile)
```

Comment:
..we can link to older docker files if you want (we just need to regen them  downloading manually from the [GCS storage repo](storage.cloud.google.com/storage.cloud.google.com/cloud-sdk-release)  where they all sit (and then calculating the hash).  probably worth providing that link in the readme.